packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -63,95 +71,126 @@ describe('AddresslistVoting', function () {
     );
     voting = await AddresslistVotingFactory.deploy();
 
+    startDate = (await getTime()) + startOffset;
+    endDate = startDate + minDuration;
+
     dao.grant(
       dao.address,
       voting.address,
       ethers.utils.id('EXECUTE_PERMISSION')
     );
     dao.grant(
       voting.address,
-      ownerAddress,
+      signers[0].address,
       ethers.utils.id('MODIFY_ADDRESSLIST_PERMISSION')
     );
   });
 
-  function initializeVoting(
-    totalSupportThresholdPct: any,
-    relativeSupportThresholdPct: any,
-    minDuration: any,
-    allowed: Array<string>
-  ) {
-    return voting.initialize(
-      dao.address,
-      totalSupportThresholdPct,
-      relativeSupportThresholdPct,
-      minDuration,
-      allowed
-    );
+  function addresslist(length: number): string[] {
+    let addresses: string[] = [];
+
+    for (let i = 0; i < length; i++) {
+      const addr = signers[i].address;
+      addresses.push(addr);
+    }
+    return addresses;
   }
 
   describe('initialize: ', async () => {
     it('reverts if trying to re-initialize', async () => {
-      await initializeVoting(1, 2, 3, []);
+      supportThreshold = pct16(50);
+      minParticipation = pct16(20);
 
-      await expect(initializeVoting(1, 2, 3, [])).to.be.revertedWith(
-        ERRORS.ALREADY_INITIALIZED
+      await voting.initialize(
+        dao.address,
+        supportThreshold,
+        minParticipation,
+        minDuration,
+        addresslist(0)
       );
+
+      await expect(
+        voting.initialize(
+          dao.address,
+          supportThreshold,
+          minParticipation,
+          minDuration,
+          addresslist(0)
+        )
+      ).to.be.revertedWith(ERRORS.ALREADY_INITIALIZED);
     });
   });
 
-  describe('Addresslisting users: ', async () => {
+  describe('Addresslisting members: ', async () => {
     beforeEach(async () => {
-      await initializeVoting(1, 2, 3, []);
+      supportThreshold = pct16(50);
+      minParticipation = pct16(20);
+
+      await voting.initialize(
+        dao.address,
+        supportThreshold,
+        minParticipation,
+        minDuration,
+        addresslist(0)
```

Comment:
Why do we need addresslist() here if the result is simply an empty array?

Reply:
We use `[]` also for other places and this way it becomes clear, that this is an empty addresslist. I can also remove it.